### DISCUSSION TITLE
Open Support Case - No Response since 7/19

### DISCUSSION THREAD

**Discussion by q3warserver** [vercel/community#3375] [Help] [answered]
### Summary

I opened a support case (00146025) and had a brief communication with a support tech back on 7/19. I was asked to provide more information which I have but I have not heard back from them. Can someone please tell me what the status is on my case? Trying to get this resolved as soon as I can.

Thank you

### Steps to Reproduce

login issue

**Comment by Icontol** [comment]
Hello, my website is thought to be phishing by Google even though there is no element of entering personal data or anything like that

**Reply by q3warserver** [reply]
Please remove this from my discussion and start your own discussion. Been waiting a while for a response and I would appreciate it if you didn't hijack my thread. Thanks. :)

**Comment by amyegan** [comment, marked as answer]
Hey @q3warserver. Thanks for flagging your case as needing more urgent attention. I’ve notified our team and it is near the top of our next cases to review. Due to the sensitive account information being shared, we’ll send all further updates through the email thread that already exists.

**Reply by q3warserver** [reply]
Thank you Amy. Vercel registration finally got back to me (thanks to you) and I was able to get my site set back up on a new account. Thanks again!
